std/algorithm/iteration.d

```diff
@@ -2695,12 +2740,110 @@ version (none) // this example requires support for non-equivalence relations
     }
 }
 
-// https://issues.dlang.org/show_bug.cgi?id=13595
-version (none) // This requires support for non-equivalence relations
+
+
+// https://issues.dlang.org/show_bug.cgi?id=13805
 @system unittest
+{
+    [""].map!((s) => s).chunkBy!((x, y) => true);
+}
+
+/**
+Like $(REF chunkBy), but does not require reflexivity, symmetricity nor
+transitivity of the binary predicate. Unlike `çhunkBy`, requires the
+source to be a forward range.
+With binary predicates, the element closer to front is always passed
+first to the predicate.
```

Comment:
```suggestion
With binary predicates, the element closer to front is always passed
as the first argument to the predicate.
```

Reply:
I quess I'm going to give another pass to the documentation anyway when I've written it to `splitBy` you suggested.